pkgs/development/python-modules/static3/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, buildPythonPackage, genshi, pytestCheckHook, pytest-cov, webtest }:
+
+buildPythonPackage rec {
+  pname = "static3";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "rmohr";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

you should just need:
```suggestion
    rev = "v${version}";
```

---

Neither is fine